library/alloc/src/collections/btree/testing/crash_test.rs

```diff
@@ -0,0 +1,123 @@
+use crate::fmt::Debug;
+use std::cmp::Ordering;
+use std::sync::atomic::{AtomicU64, Ordering::SeqCst};
+
+#[derive(Debug)]
+pub struct Scene {
+    bits_per_id: usize,
```

Comment:
Can we add some kind of comment explaining this struct?

In particular it's not clear what value should be passed for bits_per_id or what that means, but more broadly a top-level comment here describing usage would be helpful.